### PR TITLE
Remove useless npm dependencies

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue [hunk 1]
```diff
@@ -8,8 +8,9 @@
 			<li v-for="favoriteApp in favoriteApplicationsList">
 				<a :href="favoriteApp.appUrl" target="_blank">
 					<img class="myToolImage" v-if="favoriteApp.appImageFileBody != undefined && favoriteApp.appImageFileBody != ''" :src="favoriteApp.appImageFileBody"/>
-  					<span v-tooltip.bottom="favoriteApp.appDescription" class="myToolTitle">
+  					<span class="myToolTitle tooltipContent">
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue:11]: </span> missing ?

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue (hunk 1 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/vue-apps/myApplications/components/MyApplications.vue:11]: no

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**